src/templates/_includes/disclosuremenu.twig

```diff
@@ -106,7 +106,11 @@
             }|filter|keys
           } %}
             {% if item.heading is defined %}
-              <h3 class="h6 padded">{{ item.heading }}</h3>
+              {% tag 'h3' with {
```

Comment:
Instead of hardcoding `h3`, we should be able to pass different heading levels (the "Context" heading should be `h2`, while the others should be `h3`)

Reply:
done and done! You can now pass `headingTag` to an item (defaults to `h3`)